src/controllers/user-orders-controller.js

```diff
@@ -13,26 +14,15 @@ function searchSuitableDays(username, days, buffer) {
       buffer.push({ order: dayOrder, date: day.date });
     }
   });
+  return buffer;
 }
+
 export default function getOrders(username) {
-  try {
-    return new Promise((res) => {
-      const buffer = [];
-      UserOrders.find({}, (err, days) => {
-        if (err) throw err;
-        searchSuitableDays(username, days, buffer);
-        return res({
-          status: 200,
-          body: { result: buffer },
-        });
-      });
-    });
-  } catch (e) {
-    return new Promise((res) => {
+  return new Promise((res) => {
+    UserOrders.find({}).then(days =>
```

Comment:
Missing error handling. If UserOrder.find() returns an error the service will break